trie/verkle.go

```diff
@@ -213,27 +213,6 @@ func (trie *VerkleTrie) UpdateStorage(address common.Address, key, value []byte)
 }
 
 func (t *VerkleTrie) DeleteAccount(addr common.Address) error {
-	var (
-		err    error
-		values = make([][]byte, verkle.NodeWidth)
-		stem   = t.pointCache.GetTreeKeyVersionCached(addr[:])
-	)
-
-	for i := 0; i < verkle.NodeWidth; i++ {
-		values[i] = zero[:]
-	}
```

Comment:
oh I see, this code should never have made it to this branch ! :exploding_head: This is meant for the replay to simulate an account destruction. It shouldn't be there. Well, that clearly explains this bug then.